style.css

```diff
@@ -75,3 +87,309 @@ ul {
   left: 3px;
   top: 6px;
 }
+
+.works {
+  /* display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center; */
+  text-align: center;
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  align-content: center;
+  row-gap: 20px;
+}
+
+/* .work-item {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  display: grid;
+  grid-template-rows: 252px 222px;
+} */
```

Comment:
- [ ] Kindly make sure you remove the commented code lines to make your code clean and look more professional.